e2e/test/scenarios/models/models-revision-history.cy.spec.js

```diff
@@ -21,13 +21,11 @@ describe("scenarios > models > revision history", () => {
 
     openRevisionHistory();
     revertTo("You created this");
-    cy.wait("@modelQuery" + ORDERS_BY_YEAR_QUESTION_ID);
```

Comment:
No query changes - no query requests. Previous we re-run the query every time the viz settings changed